contracts/permissionless/PermissionlessOrderbookReserveLister.sol

```diff
@@ -6,12 +6,9 @@ import "../FeeBurnerInterface.sol";
 
 
 contract InternalNetworkInterface {
-    function addReserve(
-        KyberReserveInterface reserve,
-        bool isPermissionless
-    )
-        public
-        returns(bool);
+    FeeBurnerInterface public feeBurnerContract;
+    function addReserve(KyberReserveInterface reserve, bool isPermissionless) public returns(bool);
```

Comment:
why did you change it and it is not consistent with the others. please change back. also add fee burner decleration to where it used to be,